apps/examples/board_specific_demo/ameba_secure_world_demo/secure_world.c

```diff
@@ -0,0 +1,65 @@
+/****************************************************************************
+ *
+ * Copyright 2022 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ ****************************************************************************/
+
+/****************************************************************************
+ * Included Files
+ ****************************************************************************/
+
+#include <tinyara/config.h>
+#include <stdio.h>
+#include <security/security_common.h>
+#include <security/security_api.h>
+#include <tinyara/seclink.h>
+#include <tinyara/security_hal.h>
+#include <security/security_ss.h>
+#define SHOW_USAGE printf("Usage: secure_world -r -w\n");
+
+#ifdef  CONFIG_AMEBALITE_TRUSTZONE
+#define ADDR ((volatile uint32_t*)0x735C021)
+#endif
+
+/****************************************************************************
+ * secure_world_main
+ ****************************************************************************/
+
+#ifdef CONFIG_BUILD_KERNEL
+int main(int argc, FAR char *argv[])
+#else
+int secure_world_main(int argc, char *argv[])
+#endif
+{
+	int option;
+	volatile int address;
+	while (option = getopt(argc, argv, "rw")) {
+		switch (option) {
+			case 'r':
+				address = *ADDR;
+				break;
+			case 'w':
+
+				*ADDR = 1234; 
+				printf("%d",*ADDR);
+				break;
+			default:
+				SHOW_USAGE
```

Comment:
Could you please add some error message like "invalid input" before redirecting the user to the usage.